project/settings/defaults.py

```diff
@@ -227,46 +226,16 @@
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
 # base url for experiments, should be GCS bucket in prod
-EXPERIMENT_BASE_URL = os.environ.get(
-    "EXPERIMENT_BASE_URL",
-    "https://storage.googleapis.com/io-osf-lookit-staging2/experiments/",
-)  # default to ember base url
+EXPERIMENT_BASE_URL = os.environ.get("EXPERIMENT_BASE_URL")
 
 BASE_URL = os.environ.get(
     "BASE_URL", "https://localhost:8000"
 )  # default to ember base url
-OSF_URL = os.environ.get(
-    "OSF_URL", "https://staging.osf.io/"
-)  # default osf url used for oauth
 
-LOGIN_REDIRECT_URL = os.environ.get("LOGIN_REDIRECT_URL", "https://localhost:8000/exp/")
+LOGIN_REDIRECT_URL = "web:home"
 ACCOUNT_LOGOUT_REDIRECT_URL = os.environ.get("ACCOUNT_LOGOUT_REDIRECT_URL", "/api/")
```

Comment:
completely unrelated but this seems like an odd default - I guess it doesn't matter because we have the env variable set

Reply:
Yeah I saw that! Barrett Harber added it in 2017 and it hasn't been touched since ಠ_ಠ